pysat/tests/test_instrument.py

```diff
@@ -175,6 +177,23 @@ def test_prev_filename_load_default(self):
         assert (test_date == dt.datetime(2009, 1, 3))
         assert (test_date == self.testInst.date)
 
+    def test_list_files(self):
+        files = self.testInst.files.files
+        assert isinstance(files, pds.Series)
+
+    def test_remote_file_list(self):
+        files = self.testInst.remote_file_list(start=dt.datetime(2009, 1, 1),
+                                               stop=dt.datetime(2009, 1, 31))
+        assert files.index[0] == dt.datetime(2009, 1, 1)
+        assert files.index[-1] == dt.datetime(2009, 1, 31)
+
+    def test_remote_date_range(self):
+        files = self.testInst.remote_date_range(start=dt.datetime(2009, 1, 1),
+                                                stop=dt.datetime(2009, 1, 31))
+        assert len(files) == 2
```

Comment:
I'm out of power and I need to do some other stuff at the moment. It looks good I just don't understand why only 2 files here. I need to refresh my memory on remote_date_range.

Reply:
`remote_date_range` returns the first and last date on the server.